README.rst

```diff
@@ -0,0 +1,36 @@
+.. image:: docs/hstaxe/images/ACS_aXe02.png
+    :width: 400
+    :alt: HSTaXe logo
+    :align: center
+
+HSTaXe
```

Comment:
I'm not that familiar with this stuff - is there an advantage to having the README be a `.rst` file over a `.md` file?

Reply:
The main advantage is slightly for maintainability. The repo README text is straight copied from the `index` of RTD. Though unfortunately I don't think we can "import" the text, at least if we make a modification in one, it's easy to just copy-paste it to the other without having to worry about the syntax differences between `md` and `rst`